finish/src/main/java/io/openliberty/guides/bulkhead/global/eBank/microservices/Service.java

```diff
@@ -8,27 +8,23 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package global.eBank.microservices;
+package io.openliberty.guides.bulkhead.global.eBank.microservices;
```

Comment:
I don't like using the constructors to set the content differently. Let us talk about it.